requirements/common_constraints.txt

```diff
@@ -19,10 +19,10 @@ Django<2.3
 drf-jwt<1.19.1
 
 # Newer versions causing tests failures in multiple repos.
-pyjwt[crypto]==1.7.1
+
 
 # latest version requires PyJWT>=2.0.0 but drf-jwt requires PyJWT[crypto]<2.0.0,>=1.5.2
-social-auth-core<4.0.3
+
```

Comment:
`social-auth-core 4.0.2 requires PyJWT<2,>=1.4.0, but you have pyjwt 2.1.0 which is incompatible`